noetic.jsk-recognition-msgs fails to build

Fresh checkout of `develop`.

```
[jkolb@nixos:~/nix-ros-overlay]$ nix build .#noetic.jsk-recognition-msgs
error: builder for '/nix/store/7ykiqvxraby8631wr47wbchj3vlvkdxc-ros-noetic-jsk-recognition-msgs-1.2.17-r2.drv' failed with exit code 1;
       last 25 log lines:
       > This warning is for project developers.  Use -Wno-dev to suppress it.
       > 
       > -- Found PythonInterp: /nix/store/w4fvvhkzb0ssv0fw5j34pw09f0qw84w8-python3-3.11.7/bin/python (found version "3.11.7")
       > -- Using PYTHON_EXECUTABLE: /nix/store/w4fvvhkzb0ssv0fw5j34pw09f0qw84w8-python3-3.11.7/bin/python
       > -- Using default Python package layout
       > -- Found PY_em: /nix/store/mvij49mi6niiwjkqsyga5p30mjp7zhs4-python3.11-empy-4.0.1/lib/python3.11/site-packages/em.py
       > -- Using empy: /nix/store/mvij49mi6niiwjkqsyga5p30mjp7zhs4-python3.11-empy-4.0.1/lib/python3.11/site-packages/em.py
       > -- Using CATKIN_ENABLE_TESTING: OFF
       > -- catkin 0.8.10
       > -- BUILD_SHARED_LIBS is on
       > -- Using these message generators: gencpp;geneus;genlisp;gennodejs;genpy
       > -- Generating .msg files for action jsk_recognition_msgs/VQATask /build/jsk_recognition-release-release-noetic-jsk_recognition_msgs-1.2.17-2/action/VQATask.action
       > Generating for action VQATask
       > -- Generating .msg files for action jsk_recognition_msgs/ClassificationTask /build/jsk_recognition-release-release-noetic-jsk_recognition_msgs-1.2.17-2/action/ClassificationTask.action
       > Generating for action ClassificationTask
       > -- jsk_recognition_msgs: 90 messages, 24 services
       > CMake Error at CMakeLists.txt:145 (if):
       >   if given arguments:
       >
       >     "STREQUAL" "indigo"
       >
       >   Unknown arguments specified
       >
       > 
       > -- Configuring incomplete, errors occurred!
       For full logs, run 'nix log /nix/store/7ykiqvxraby8631wr47wbchj3vlvkdxc-ros-noetic-jsk-recognition-msgs-1.2.17-r2.drv'.
```

--- a/distros/distro-overlay.nix
+++ b/distros/distro-overlay.nix
@@ -66,9 +66,12 @@ let
     image-cb-detector = patchBoostSignals rosSuper.image-cb-detector;
 
     jsk-recognition-msgs = rosSuper.jsk-recognition-msgs.overrideAttrs ({
-      buildInputs ? [], ...
+      buildInputs ? [], postPatch ? "", ...
     }: {
       buildInputs = buildInputs ++ [ rosSuper.ros-environment ];
+      postPatch = ''
+        substituteInPlace CMakeLists.txt --replace "catkin_python_setup()" ""
+      '';
     });
 
     laser-cb-detector = patchBoostSignals rosSuper.laser-cb-detector;

--- a/distros/distro-overlay.nix
+++ b/distros/distro-overlay.nix
@@ -65,6 +65,12 @@ let
 
     image-cb-detector = patchBoostSignals rosSuper.image-cb-detector;
 
+    jsk-recognition-msgs = rosSuper.jsk-recognition-msgs.overrideAttrs ({
+      buildInputs ? [], ...
+    }: {
+      buildInputs = buildInputs ++ [ rosSuper.ros-environment ];
+    });
+
     laser-cb-detector = patchBoostSignals rosSuper.laser-cb-detector;
 
     libfranka = rosSuper.libfranka.overrideAttrs ({

